### DISCUSSION TITLE
Examples to test multi-cluster support

### DISCUSSION THREAD

**Discussion by donovanmuller** [keptn/keptn#3414] [Q&A] [answered]
The release notes for [`0.8.0-rc1`](https://github.com/keptn/keptn/releases/tag/0.8.0-rc1) mention multi-cluster support. However, I can't find any mention of how to test this new feature?

Are there any examples to test multi-cluster support?

To be more specific:

* What are the steps to installing just the execution plane in the "workload" clusters?
* How do you reference which cluster a deployment should be applied too?

**Comment by donovanmuller** [comment]
There is a reference to the [execution plane](https://keptn.sh/docs/0.8.x/operate/k8s_support/#execution-plane) in the "Kubernetes support & Cluster size" [section](https://keptn.sh/docs/0.8.x/operate/k8s_support/) in the docs.

**Comment by donovanmuller** [comment, marked as answer]
After the `0.8.0` release this is now described in the docs under ["Multi-cluster setup"](https://keptn.sh/docs/0.8.x/operate/multi_cluster/)

**Reply by johannes-b** [reply]
Hi, 
I'm very sorry that we overlooked this discussion. 
You are right, the docu for setting up a multi-cluster environment by splitting the control- from the execution-plane is provided by the link you posted. (keptn.sh/docs > Release 0.8 > Operate Keptn > [Multi-cluster setup](https://keptn.sh/docs/0.8.x/operate/multi_cluster/) ) 

If you have ideas for improving the docu, please let me know.
